plugins/org.eclipse.n4js/META-INF/MANIFEST.MF

```diff
@@ -35,7 +35,8 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  com.fasterxml.jackson.core.jackson-databind,
  org.eclipse.n4js.external.libraries,
  org.eclipse.core.jobs,
- org.eclipse.xtext.builder
+ org.eclipse.xtext.builder,
+ org.eclipse.core.resources;bundle-version="[3.11.0,4.0.0)"
```

Comment:
I was adding this during testing different things and this might be obsolete now.

I understand that the version in the targetplatform is computed using also the manifest files, and thus, at some bundles, versions might be necessary.